src/material/form-field/form-field.ts

```diff
@@ -507,7 +507,8 @@ export class MatFormField extends _MatFormFieldMixinBase
     if (this._control) {
       let ids: string[] = [];
 
-      if (this._control.userAriaDescribedBy) {
+      if (this._control.userAriaDescribedBy &&
+        this._control.userAriaDescribedBy.split) {
```

Comment:
Can we do a `typeof` check here instead? It also needs a comment to explain why it wouldn't be a string as required by the type

Reply:
Fixed and I checked that this change works internally and it does